### PR TITLE
Set up the expectation of a paid service

### PR DESCRIPTION
**Pull request by pbrisbin**

### CODE REVIEW

File: templates/docs.hamlet [hunk 1]
```diff
@@ -142,12 +142,20 @@
   <h2>Support
 
   <p>
+
+    The Carnival hosted service is currently in Beta. We'd love for you to sign
+    up and give it a try to help us make a great product. It will eventually be
+    a paid service.
```
Comment (jferris) [inline templates/docs.hamlet]: Do we want to add something like, "but for now, you can try it for free?"

**Comment by jferris** [comment]
This seems good to me. Maybe we should get the copyediting crew to review the site in general?

**Comment by pbrisbin** [comment]
Do you think anything should change here knowing that #195 is imminent?

**Comment by jferris** [comment]
I think it's fine to adjust whatever we need to in #195 after this is merged.

**Comment by pbrisbin** [comment]
Who's the copy-editing crew? This was suggested by @cpytel

**Comment by jferris** [comment]
> Who's the copy-editing crew?

I think I've seen @gabebw, @mike-burns, @adarsh, and @croaky review copy most often. I just created a @thoughtbot/copy-editing team for this kind of thing.

**Comment by gabebw** [comment]
This looks reasonable.

**Pull request merged**